corehq/apps/cloudcare/static/cloudcare/js/form_entry/fullform-ui.js

```diff
@@ -277,8 +277,10 @@ function Form(json) {
     };
 
     self.afterRender = function() {
-        // Once form has finished rendering, render all popovers
-        $('.js-form-container [data-toggle="popover"]').popover();
+        $(".help-text-trigger").click(function(event) {
+            var container = $(event.target).closest(".caption");
```

Comment:
Very picky, but it's good to get into the habit of using `currentTarget` instead of `target`. `currentTarget` is the element the handler is attached to, whereas `target` is the element that actually received the event (may be a child of `currentTarget`). This code will work either way, just a best practice.